### PR TITLE
Anders test

### PR DESCRIPTION
**Pull request by verysecrethero**
Boost Tests has been added: ConfigFileTest, EventTest, InputManagerTest ("empty"), OctTree, ResourceManagerTest

### CODE REVIEW

**Comment by verysecrethero** [comment]
Dealt with the Exit Crash

**Comment by sippeangelo** [comment]
Revert 0ae6c42

**Comment by verysecrethero** [comment]
Im adding some Health-related updates to this commit

File: include/Engine/Core/EPlayerDeath.h [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+#include "EventBroker.h"
+#include "../Core/Entity.h"
+
+namespace Events
+{
+
+struct PlayerDeath : Event
+{
+    std::string KilledBy;
```
Comment (wima12) [inline include/Engine/Core/EPlayerDeath.h]: EntityID here instead of std::string?

**Pull request merged**